css/templates/travel/elements/_btn.css

```diff
@@ -24,20 +24,20 @@
  }
 
 .ampstart-btn[disabled] {
-  opacity: 0.5;
-  cursor: default;
+opacity: 0.5;
```

Comment:
wrong indents , i think you did not intend to modify this file

Reply:
done